LDM-148.tex

```diff
@@ -992,30 +993,38 @@ \subsubsection{Template Generation
 Data Release Productions. Uses raw science images to generate the
 templates, placing them in the Data Backbone.
 
-\subsubsection{Annual Calibration Products Production
-Payload}\label{annual-calibration-products-production-payload}
-
-Executes at annual intervals prior to the start of the Data Release Production. Uses
-raw calibration images, information from the Transformed EFD,
-information from the Auxiliary Telescope Spectrograph, and external
-catalogs to generate Master Calibration Images and Calibration Database
-entries in the Data Backbone.
-
-\subsubsection{Data Release Production
-Payload}\label{data-release-production-payload}
-
-Executes at annual intervals,
-first running a ``mini-DRP'' over a small portion of the sky, followed
-by the full DRP over the entire sky. Produces science data products in
-the Data Backbone.
-
-\subsection{Firefly and SUIT}\label{suit}
+\subsection{SUIT}\label{suit}
 
 The Science User Interface and Tools, primarily composed of the Firefly product, provide visualization, plotting,
 catalog rendering, browsing, and searching elements that can be
 assembled into predetermined ``portals'' but can also be used flexibly
 within dynamic ``notebook'' environments.
 
+\subsection{Supporting Software}\label{supporting-software}
+
+These software products, plus the middleware in the nextt subsection, are key underlying components to services and other software products.
+
+\subsubsection{Science Pipelines Libraries}\label{science-pipelines-libraries}
+
+Provide objects, data structures, algorithms, and frameworks for writing Science Payloads.
+These libraries contain the key scientific analysis routines used to perform processing and measurement to generate the LSST data products.
+
+\subsubsection{Distributed Database
```

Comment:
I think it would be helpful to be more specific about where Qserv is actually running.

The changes in §4 mean that it is now listed as “infrastructure” component, but there's no service in §12 which seems to be devoted to running it; it's not clear how that designation as “infrastructure” above is supposed to relate to designation as “supporting software” here.

Reply:
It says it is part of the catalog data access web service, and hence part of the LSP, but I will clarify.